src/modules/itemCount.js

```diff
@@ -0,0 +1,15 @@
+import { seriesApiUrl } from './api.js';
+
+const itemCounter = (item) => {
+  const totalCount = document.getElementById('total-count');
+  totalCount.innerHTML = `(${item})`;
+};
```

Comment:
- Kindly make sure to follow the guidelines in the card `Add all items counter on the Homepage - Student A` in this [issues](https://github.com/zuhebahmed88091/ShowAddit_capstone_js/issues/6): 
   - [ ] A counter function should look for specific DOM elements. 
   - [ ] A counter function should cover all the edge cases you can think about.